osu.Game/Screens/Play/PlayerSettings/BeatmapOffsetControl.cs

```diff
@@ -320,6 +349,21 @@ public void OnReleased(KeyBindingReleaseEvent<GlobalAction> e)
         {
         }
 
+        private static double computeSuggestedOffset(double median, double unstableRate, double currentOffset, bool proportionalToUnstableRate)
+        {
+            const double ur_adjustment_cutoff = 90;
+            const double exponential_factor = -0.0116;
+
+            double offsetAdjustment = median;
+
+            if (proportionalToUnstableRate && unstableRate >= ur_adjustment_cutoff)
```

Comment:
This is super weird. A global setting changing the algorithm used.

If we're semi-confident about the new UR basis, then let's just apply it everywhere in all cases.

cc @bdach may have thoughts on this.